service/resource/vaultrolev1/resource.go

```diff
@@ -8,7 +8,10 @@ import (
 )
 
 const (
-	Name = "vaultrolev1"
+	// AllowSubdomains defines whether to allow the generated root CA of the PKI
+	// backend to allow sub domains as common names.
+	AllowSubdomains = true
```

Comment:
This seems to be the standard here. We do not configure it via CROs.

Reply:
I don't see a reason to extract a constant. I'd just describe variable inside a function to not jump around.

Also I would make those constants private.

Reply:
I always like this for godoc reasons. When you check the package you directly see that it is the hardcoded default.